README.md

```diff
@@ -508,3 +504,15 @@ open erd.pdf
 ```
 
 The command uses the [rails-erd](https://github.com/voormedia/rails-erd) gem.
+
+### Updating the table of contents
+
+To update the table of contents after adding things to this README you can use the [markdown-toc](https://github.com/jonschlinkert/markdown-toc)
+node module. To install it, make sure you have installed the dev dependencies from yarn. To update
+the table of contents, run:
```

Comment:
It's better to keep each sentence on its own line.  This makes diffs in future edits much easier to parse and relies on line breaks being after logical units of text and not because of some abstract column count.  Git relies on line breaks not moving around too much for its diffs to be useful.

```suggestion
To update the table of contents after adding things to this README you can use the [markdown-toc](https://github.com/jonschlinkert/markdown-toc) node module.
To install it, make sure you have installed the dev dependencies from yarn.
To update the table of contents, run:
```